meta-iot2000-bsp/recipes-kernel/linux/linux-cip_4.4.inc

```diff
@@ -3,12 +3,12 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=d7810fab7487fb0aad327b76f1be7cd7"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/configs:${THISDIR}/patches:"
 
-LINUX_VERSION = "4.4.238-cip50"
+LINUX_VERSION = "4.4.302-cip69"
```

Comment:
FWIW: There will likely be -cip70 around next week. If we can wait, we could jump even there.

Reply:
What would be the benefits of picking -cip70 compare to -cip69? If nothing big then I think we can stay on -cip69